internal/controller/volumereplicationgroup_controller.go

```diff
@@ -881,7 +881,7 @@ func (v *VRGInstance) separatePVCUsingPeerClassAndSC(peerClasses []ramendrv1alph
 				return nil
 			}
 
-			return fmt.Errorf("failed to find matching peerClass for PVC") // TODO: better error message
+			return fmt.Errorf("failed to find replicationClass matching peerClass for PVC")
```

Comment:
Maybe it is good to add PVC name as well (if it is not redundant somewhere else)?

Reply:
done.